dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/injection/EEValidationUtils.java

```diff
@@ -212,26 +212,27 @@ private static void validateJndiLookup(InitialContext c, String lookupString, An
                 try {
                     String className = pair.getClassName();
                     Class<?> jndiClass = cdiArchive.getClassLoader().loadClass(className);
-                    if ("javax.resource.cci.ConnectionFactory".equals(className)) {
-                        try {
-                            Object o = c.lookup(lookupName);
-                            if (o != null) {
-                                jndiClass = o.getClass();
-                            }
-                        } catch (RuntimeException e) {
-                            // An error occurred while getting the object from JNDI. This may happen
-                            // at this early point in the initialisation process, but if so we just
-                            // skip validation.
-                        }
-                    } else {
-                        try {
-                            jndiClass = cdiArchive.getClassLoader().loadClass(className);
-                        } catch (ClassNotFoundException ex) {
-                            // Couldn't load the jndiClass name, can't validate
-                        }
+                    try {
```

Comment:
I expect that you intended to remove the line above?

Reply:
good point, I didn't realize this was redundant with the try/catch block. I can just remove the try/catch entirely since we're already doing `cdiArchive.getClassLoader().loadClass(className)` earlier with no try/catch